nucypher/network/server.py

```diff
@@ -258,7 +258,7 @@ def set_policy(id_as_hex):
         kfrag = KFrag.from_bytes(kfrag_bytes)
 
         if not kfrag.verify(signing_pubkey=alices_verifying_key):
-            raise InvalidSignature("{} is invalid".format(kfrag))
+            return Response(f"Signature on {kfrag} is invalid", status=403)
```

Comment:
Should we also do something similar on L265 where we raise `alice.SuspiciousActivity`?

Reply:
Yes that’s a good idea.  Will add it now.